plugins/module_utils/network/sonic/config/system/system.py

```diff
@@ -395,6 +411,7 @@ def get_replaced_config(self, have, want):
 
         w_hostname = want.get('hostname', None)
         w_intf_name = want.get('interface_naming', None)
+        w_auto_breakout_mode = want.get('auto_breakout', None)
```

Comment:
Should line#416 be added with "or  w_auto_breakout_mode is not None"?

Reply:
> Should line#416 be added with "or w_auto_breakout_mode is not None"?

Yes that had to be added. Addressed it. Thank you for pointing it out.